service-bus.api.md

```diff
@@ -0,0 +1,337 @@
+## API Report File for "@azure/service-bus"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { AmqpMessage } from "@azure/core-amqp";
+import { delay } from "@azure/core-amqp";
+import { Delivery } from "rhea-promise";
+import Long from "long";
+import { MessagingError } from "@azure/core-amqp";
+import { OperationOptions } from "@azure/core-auth";
+import { RetryOptions } from "@azure/core-amqp";
+import { TokenCredential } from "@azure/core-amqp";
+import { TokenType } from "@azure/core-amqp";
+import { WebSocketImpl } from "rhea-promise";
+import { WebSocketOptions } from "@azure/core-amqp";
+
+// @public
+export interface CorrelationFilter {
+  contentType?: string;
+  correlationId?: string;
+  label?: string;
+  messageId?: string;
+  replyTo?: string;
+  replyToSessionId?: string;
+  sessionId?: string;
+  to?: string;
+  userProperties?: any;
+}
+
+// @public
+export interface CreateBatchOptions extends OperationOptions {
+  maxSizeInBytes?: number;
+}
+
+// @public
+export interface DeadLetterOptions {
+  deadLetterErrorDescription: string;
+  deadLetterReason: string;
+}
+
+export { delay };
+
+export { Delivery };
+
+// @public
+export interface GetMessageIteratorOptions extends OperationOptions, WaitTimeOptions {}
+
+// @public
+export interface GetReceiverOptions {
+  retryOptions?: RetryOptions;
```

Comment:
The theory that one would want to have retry options for a sender/receiver that is different from the ones set on the client is not a proved one. This should be something we add when we have user feedback for the same. Since this can be additional, we can always add it back.